.github/workflows/tests.yml

```diff
@@ -628,6 +628,10 @@ jobs:
       - name: install harbor-operator
         run: |
           set -ex
+          make kustomize
+
+          ./bin/kustomize build --reorder legacy config/helm/crds | kubectl create -f -
```

Comment:
Shouldn't we add a Makefile target to install crds ?

Reply:
This is how it is done in the other tests currently - I am happy to create an issue to align this and not do this crazy stuff anymore :D